inventory/src/components/ItemList/ViewExistingBlueprintTable.js

```diff
@@ -3,84 +3,85 @@ import {
   Accordion,
   AccordionSummary,
   AccordionDetails,
-  Paper,
-  Table,
-  TableBody,
-  TableCell,
-  TableContainer,
-  TableHead,
-  TableRow,
+  Divider,
+  Grid,
+  Stack,
   Box,
   Typography,
 } from '@mui/material';
 import React from 'react';
 export const ViewExistingBlueprintTable = ({ items }) => {
   return (
     <Box className='dynamic-width'>
-      <TableContainer component={Paper} style={{ overflow: 'scroll' }}>
-        <Table>
-          <TableHead>
-            <TableRow key='View_Blueprint_Header'>
-              <TableCell align='center'>
-                <h3>Blueprints</h3>
-              </TableCell>
-            </TableRow>
-          </TableHead>
-          <TableBody>
-            {items.map((item) => (
-              <Accordion
-                key={`Blueprint ${item.id}`}
-                className='view-existing-blueprints'
-              >
-                <AccordionSummary
-                  data-testid='item_accordion'
-                  expandIcon={<ExpandMoreIcon />}
-                >
-                  <TableCell>
-                    <Typography data-testid='item_name' variant={'h6'}>
-                      {item.name}
-                    </Typography>
-                  </TableCell>
-                  <TableCell align={'center'}>{item.quantity}</TableCell>
-                </AccordionSummary>
-                <AccordionDetails>
-                  <TableContainer
-                    component={Paper}
-                    style={{ overflow: 'scroll' }}
+      <Accordion expanded={false} className='view-existing-blueprints-header'>
+        <Box className='view-existing-blueprints-subheader'>
+          <Typography variant='h6'>Blueprints</Typography>
+        </Box>
+      </Accordion>
+      <Accordion
+        expanded={false}
+        className='view-existing-blueprints-header'
+        sx={{ marginBottom: 1 }}
+      >
```

Comment:
```suggestion
      <Accordion
        expanded={false}
        className='view-existing-blueprints-header'
        sx={{ mt: 3 }}
      >
```

Remove "Blueprints" header and bottom margin, and add a top margin